test/functional/search_api_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class SearchApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  test 'search notes functionality' do
+    get '/api/srch/notes?srchString=Blog'
+    assert last_response.ok?
+
+    # Expected search pattern
+    pattern = {
+      srchParams: {
+        srchString: 'Blog',
+        seq: nil,
+      }.ignore_extra_keys!
+    }.ignore_extra_keys!
+
+    matcher = JsonExpressions::Matcher.new(pattern)
+
+    puts JSON.parse(last_response.body)
```

Comment:
Also, we can now remove this debugging line again, now that we've captured the output and know what it's supposed to look like. Thanks!